sdk/storagesync/Azure.ResourceManager.StorageSync/api/Azure.ResourceManager.StorageSync.netstandard2.0.cs

```diff
@@ -349,13 +361,15 @@ protected StorageSyncServiceCollection() { }
     public partial class StorageSyncServiceData : Azure.ResourceManager.Models.TrackedResourceData, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.StorageSync.StorageSyncServiceData>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.StorageSync.StorageSyncServiceData>
     {
         public StorageSyncServiceData(Azure.Core.AzureLocation location) { }
+        public Azure.ResourceManager.Models.ManagedServiceIdentity Identity { get { throw null; } set { } }
         public Azure.ResourceManager.StorageSync.Models.IncomingTrafficPolicy? IncomingTrafficPolicy { get { throw null; } set { } }
         public string LastOperationName { get { throw null; } }
         public string LastWorkflowId { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.StorageSync.StorageSyncPrivateEndpointConnectionData> PrivateEndpointConnections { get { throw null; } }
         public string ProvisioningState { get { throw null; } }
         public int? StorageSyncServiceStatus { get { throw null; } }
         public System.Guid? StorageSyncServiceUid { get { throw null; } }
+        public bool? UseIdentity { get { throw null; } }
```

Comment:
Consider renaming to `IsUseIdentity`.

Reply:
I think `is` is redundant here, Use is a verb already

Reply:
we have a policy to have `Is` with `adjective` for bool type state, like `IsIdentified`, which is not the case here

Reply:
`DoesUseIdentity` should be `UsesIdentity`, I think it should be fine to use normal verb form here